ports/aliyun-oss-cpp-sdk/unofficial-aliyun-oss-cpp-sdk-config.cmake

```
@@ -0,0 +1,28 @@
+find_path(ALIYUN_OSS_CPP_SDK_INCLUDE_DIR NAMES alibabacloud/oss/OssClient.h PATHS "${_VCPKG_INSTALLED_DIR}/${VCPKG_TARGET_TRIPLET}/include" NO_DEFAULT_PATH)
+find_library(ALIYUN_OSS_CPP_SDK_LIBRARY_RELEASE NAMES alibabacloud-oss-cpp-sdk PATHS "${_VCPKG_INSTALLED_DIR}/${VCPKG_TARGET_TRIPLET}/lib" NO_DEFAULT_PATH)
+find_library(ALIYUN_OSS_CPP_SDK_LIBRARY_DEBUG   NAMES alibabacloud-oss-cpp-sdk PATHS "${_VCPKG_INSTALLED_DIR}/${VCPKG_TARGET_TRIPLET}/debug/lib" NO_DEFAULT_PATH)
```

Allow users to set CMAKE_PREFIX_PATH

```suggestion
find_path(ALIYUN_OSS_CPP_SDK_INCLUDE_DIR NAMES alibabacloud/oss/OssClient.h PATHS "include")
find_library(ALIYUN_OSS_CPP_SDK_LIBRARY_RELEASE NAMES alibabacloud-oss-cpp-sdk PATHS "lib")
find_library(ALIYUN_OSS_CPP_SDK_LIBRARY_DEBUG   NAMES alibabacloud-oss-cpp-sdk PATHS "debug/lib")
```